src/components/MultiStepIndicator/MultiStepIndicator.jsx

```diff
@@ -81,7 +108,14 @@ MultiStepIndicator.propTypes = {
   /** For overriding the 'fulfilled' step's icon. Is passed directly to an Icon component. */
   fulfilledStepIcon: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
   /** For overriding the 'fulfilled' step's icon type. Necessary when passing a string in the "fulfilledStepIcon" prop. */
-  fulfilledStepIconType: PropTypes.oneOf([Icon.type.SVG, Icon.type.ICON_FONT])
+  fulfilledStepIconType: PropTypes.oneOf([Icon.type.SVG, Icon.type.ICON_FONT]),
+  /** Callback for clicking each step. The callback is sent one parameter - the step's number. */
+  onClick: PropTypes.func,
+  /** Determines the step's text placement. */
```

Comment:
I'd elaborate a bit more - something like "Determines the steps' texts' placement: either next to the indicators (horizontal placement), or below them (vertical)."